python/tvm/relay/frontend/tensorflow.py

```diff
@@ -914,10 +914,11 @@ def _impl(inputs, attr, params, mod):
 
 
 def _sparse_tensor_dense_matmul():
-    # Sparse utility from scipy
-    from scipy.sparse import csr_matrix
-
     def _impl(inputs, attr, params, mod):
+        # Loading this by default causes TVM to not be loadable from other languages.
```

Comment:
Just curious, what kind of error does loading scipy cause for other bindings?

Reply:
There use to be a bad initialization error with SciPy I think this is not out of date due to lack auto-initialization